.github/workflows/release.yml

```
@@ -0,0 +1,134 @@
+name: Release
+
+on:
+  push:
+    branches:
+        - main
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release'
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: read
+  statuses: write
+  packages: write
+
+jobs:
+  release:
+    name: Release Tuist App
+    runs-on: macos-latest
+    timeout-minutes: 50
+    if: ${{ !contains(github.event.head_commit.message, '[Release]') }}
```

Should we make the prefix more specific?

```suggestion
    if: ${{ !contains(github.event.head_commit.message, '[Release] Tuist App') }}
```